source/deployment-options/wazuh-from-sources/wazuh-agent/index.rst

```diff
@@ -41,6 +41,7 @@ The Wazuh agent is a single and lightweight monitoring software. It is a multi-p
                             .. code-block:: console
 
                                 # curl -OL https://packages.wazuh.com/utils/cmake/cmake-3.18.3.tar.gz && tar -zxf cmake-3.18.3.tar.gz
+                                # curl -OL http://packages.wazuh.com/utils/gcc/gcc-9.4.0.tar.gz && tar xzf gcc-9.4.0.tar.gz  && cd gcc-9.4.0/ && ./contrib/download_prerequisites && ./configure --enable-languages=c,c++ --prefix=/usr --disable-multilib --disable-libsanitizer && make -j$(nproc) && make install && ln -fs /bin/g++ /usr/bin/c++ && ln -fs /bin/gcc /usr/bin/cc && cd .. && rm -rf gcc-*
```

Comment:
Is this line misplaced? This is the CentOS 6/7 tab and commands for CentOS8 are found below. We're updating commands for CentOS8 (as per the title and description of this PR).

Reply:
Hi Javi! We need to add the step of GCC for Centos8 too.

Reply:
Exactly. Now it seems to be misplaced in cmake installation lines rather than in CentOS 8 tab

![image](https://github.com/wazuh/wazuh-documentation/assets/47069802/8a0f498d-d5d0-4ff0-9db3-9d24452078fa)

Reply:
Oh, I see. Sorry, fixed!